src/copy.service.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const fs = require('fs');
+
+const copy = (from, to) => {
+  try {
+    const data = fs.readFileSync(from);
```

Comment:
This way you are loading the **entire** file into memory.
E.g program will be inefficient if you copy files that are several GBs in size

Use streams to avoid this